Editor/UI/Drawers/ConditionInstantiatorDrawer.cs

```diff
@@ -25,6 +25,10 @@ public override Rect Draw(Rect rect, object currentValue, Action<object> changeV
             }
             EditorGUI.EndDisabledGroup();
 
+            if (EditorDrawingHelper.DrawHelpButton(ref rect))
+            {
+                Application.OpenURL("https://developers.innoactive.de/documentation/creator/latest/articles/innoactive-creator/default-conditions.html");
```

Comment:
This button in my opinion seems like over communication and more disturbing than helpful. What is confusing about "Add Condition"?

Reply:
the point is that you need help BEFORE adding the behaviour or condition, because 'what does snap object mean?'  Let's try it, can quickly be removed  in the future.